exporter/stats/stackdriver/stackdriver.go

```diff
@@ -299,6 +303,14 @@ func newTypedValue(view *stats.View, r *stats.Row) *monitoringpb.TypedValue {
 		return &monitoringpb.TypedValue{Value: &monitoringpb.TypedValue_Int64Value{
 			Int64Value: int64(*v),
 		}}
+	case *stats.SumData:
+		return &monitoringpb.TypedValue{Value: &monitoringpb.TypedValue_DoubleValue{
+			DoubleValue: float64(*v),
+		}}
+	case *stats.MeanData:
+		return &monitoringpb.TypedValue{Value: &monitoringpb.TypedValue_DoubleValue{
+			DoubleValue: v.Mean,
```

Comment:
A cumulative metric containing just the mean isn't going to work very well.  I suggest converting the MeanData to a distribution.  (You could also use a gauge metric, but aggregating across timeseries wouldn't work as well.)

Reply:
Make sense to me. Also updated in https://github.com/census-instrumentation/opencensus-go/pull/224